README.md

```diff
@@ -1,4 +1,52 @@
-[![GitHub Actions CI/CD build status — Collection test suite](https://github.com/ansible-collection-migration/community.general/workflows/Collection%20test%20suite/badge.svg?branch=master)](https://github.com/ansible-collection-migration/community.general/actions?query=workflow%3A%22Collection%20test%20suite%22)
+ [![Shippable build status](https://api.shippable.com/projects/5e664a167c32620006c9fa50/badge?branch=master)](https://app.shippable.com/projects/5e664a167c32620006c9fa50)
 
-Ansible Collection: community.general
-=================================================
\ No newline at end of file
+# Ansible Collection: community.general
+
+This repo contains the `community.general` Ansible Collection.
+
+The collection includes the modules and plugins supported by Ansible community.
+
+
+## Installation and Usage
+
+### Installing the Collection from Ansible Galaxy
+
+Before using the General community collection, you need to install the collection with the `ansible-galaxy` CLI:
+
+    ansible-galaxy collection install community.general
+
+You can also include it in a `requirements.yml` file and install it via `ansible-galaxy collection install -r requirements.yml` using the format:
+
+```yaml
+collections:
+- name: community.general
+```
+
+## Testing and Development
+
+If you want to develop new content for this collection or improve what is already here, the easiest way to work on the collection is to clone it into one of the configured [`COLLECTIONS_PATHS`](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths), and work on it there.
+
+
+### Testing with `ansible-test`
+
+TBD
+
+## Publishing New Version
+
+TBD
+
+## More Information
+
+TBD
+
+## Communication
+
+We have a dedicated Working Group for Ansible development.
+You can find other people interested in this in `#ansible-devel` on Freenode IRC.
```

Comment:
probably worth mention that there's at least 2 channels, i.e. #ansible-devel and #ansible-community

Reply:
Good spot, thanks for suggesting this.